### ISSUE TITLE
This is almost certainly illegal

### ISSUE THREAD

**Issue by Czar-Vili** [dadsquatch/f5oclock#31]
I'm not sure if you have checked with a lawyer, but scraping a news article and then showing ads around it is almost certainly illegal. If you don't show ads you might be ok, but making a profit off another organizations work with out providing them a share of the profit is probably frowned upon.

**Comment by dadsquatch** [comment]
Summarizing articles is not illegal. We link to the full source. The only data we use is whats put in their meta tags for sharing and aggrigating.



Sent from my Verizon, Samsung Galaxy smartphone


-------- Original message --------

**Comment by Czar-Vili** [comment]
welp you guys have probably done your research so I wish you the best of luck! I love the site and don't want to see it running into any issues.

**Comment by dadsquatch** [comment]
You might be right though. TLDR is going to be replaced anyways down the road with better things. We are going tondisable it rhis weekend and make the site a bit more user friendly.



Sent from my Verizon, Samsung Galaxy smartphone


-------- Original message --------

**Comment by dadsquatch** [comment]
TLDR has been removed.
